packages/atomic/src/components/common/button.spec.ts

```diff
@@ -75,109 +65,110 @@ describe('button', () => {
       onClick: handleClick,
     };
 
-    const button = renderButton(props);
+    const button = await renderButton(props);
 
-    await fireEvent.click(button);
+    await userEvent.click(button);
```

Comment:
90% sure it's the same thing behind the scenes, but well,l "do NOT cross the streams"

Reply:
Yes, `userEvent` should be user instead of `fireEvent`.
However, another way to do it could be to target the locator
```ts
const button = await page.getByRole('button');
await button.click();
```